extensions/fairifier-rdf-extension/dtls/fairifier/LoadRdfSkeletonCommand.java

```diff
@@ -20,14 +29,49 @@ public void doPost(HttpServletRequest req, HttpServletResponse res) throws Servl
         res.setCharacterEncoding("UTF-8");
         res.setHeader("Content-Type", "application/json");
         JSONWriter writer = new JSONWriter(res.getWriter());
-        try {
-            writer.object();
-            writer.key("skeleton");         
-            String out = rdfSkeletonService.loadModel(req.getParameter("projectId"));
-            writer.value(out);
-            writer.endObject();
-        }catch(JSONException e) {
+        try {        
+            Project project = getProject(req); //this needs the project parameter, indicating the project on which the skeleton should be applied
+            
+            String jsonString = req.getParameter("schema");
+            JSONObject json = ParsingUtilities.evaluateJsonStringToObject(jsonString);
+            RdfSchema schema = RdfSchema.reconstruct(json);
+            
+            AbstractOperation op = new LoadRdfSchemaOperation(schema);
+            Process process = op.createProcess(project, new Properties());
+            
+            performProcessAndRespond(req, res, project, process);
+            
+        }catch(Exception e) {
             respondException(res, e);
         }
     }
+    
+    static protected void performProcessAndRespond(
```

Comment:
Could you explain why this method is static?

Reply:
It is a overloaded method from the Command class which is static

Reply:
Java doesn't allow you to override static members from a parent class. So I'd propose to either make this a private non-static method and rename it, or invoke the parent method (if it is applicable) and do custom logic after the invocation.